dm/dm/master/config.go

```diff
@@ -91,7 +91,7 @@ func NewConfig() *Config {
 }
```

Comment:
--openapi for command line config?

Reply:
now i think config file is enough becasue we will enable OpenAPI as default in the feature

Reply:
I don't think so.